src/main/java/com/enterprise/fastfoodapplication/dto/ShoppingCart.java

```diff
@@ -11,15 +13,11 @@
     private String shoppingId;
     private Map<String, CartOrder> orders; //We can probably use foodId as key in this Map object
 
-    public ShoppingCart() {
-        orders = new HashMap<String, CartOrder>();
-    }
-
-    public ShoppingCart(String shoppingId) {
-        this.shoppingId = shoppingId;
-        orders = new HashMap<String, CartOrder>();
-    }
 
+    /**
+     *
+     * @return
+     */
```

Comment:
Only add JavaDoc if it is providing value over and above what can be read from the method.